webpack/scenes/ContentViews/Details/Filters/AffectedRepositories/AffectedRepositoryTable.js

```diff
@@ -197,6 +200,7 @@ const AffectedRepositoryTable = ({
         updateSearchQuery,
         activeFilters,
         defaultFilters,
+        resetFilters,
```

Comment:
What is this doing?

Reply:
I was getting [this error](https://github.com/Katello/katello/blob/5bc66c29aa1c3674ee432762a071394c11400666/webpack/components/Table/EmptyStateMessage.js#L151) at one point on one of the tests.